scripts/download_hash.sh

```diff
@@ -139,7 +139,7 @@ function get_checksums() {
 ["gvisor_runsc_binary"]="arm ppc64le"
 ["gvisor_containerd_shim_binary"]="arm ppc64le"
 ["containerd_archive"]="arm"
-["skopeo_binary"]="arm ppc64le"
+["skopeo_binary"]="arm"
 )
```

Comment:
HI @Jainbrt 
why the `ppc64le` be removed 🤔?

Reply:
HI @yankay ppc64le is not removed, while this enabled us to add the automatic checksum for ppcle64 arch in file `roles/kubespray-defaults/defaults/main/checksums.yml` for skopeo binary.